cli/generators/component/templates/component.js

```diff
@@ -0,0 +1,26 @@
+import React, {PropTypes} from 'react';
```

Comment:
Good question. As you noticed is not being used in the template but: if we are going to prefer adding `propTypes` and `defaultProps` because 83% of the components uses it then `PropTypes` is going to be used and it makes sense to add it.

But that question is why I don't like code generation too much hehe. Is a gray middle-ground. What do you think?